chord_metadata_service/discovery/utils.py

```diff
@@ -242,9 +242,14 @@ def get_public_model_scoped_queryset(scope: ValidatedDiscoveryScope, mn: PublicM
         filter_scope = "project"
         value = scope.project_id
     else:
-        return PUBLIC_MODEL_NAMES_TO_MODEL[mn].objects.all()
+        return PUBLIC_MODEL_NAMES_TO_MODEL[mn].objects.distinct()
```

Comment:
Not sure why distinct is needed here?

Reply:
see https://redmine.c3g-app.sd4h.ca/issues/2365 - i'll add a comment, but otherwise nested filters create joins which make duplicate rows